keyboards/gmmk/pro/mcuconf.h

```diff
@@ -0,0 +1 @@
+#include_next "mcuconf.h"
```

Comment:
If this isn't used, this file can be removed, and it will default to the default config.